web/pandas/versions.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "1.5 (dev)",
+        "version": "docs/dev"
+    },
+    {
+        "name": "1.4.0 (rc)",
+        "version": "pandas-docs/version/1.4"
+    },
+    {
+        "name": "1.3.5 (stable)",
+        "version": "docs"
```

Comment:
What's the difference between this (stable) and pandas-docs/version/1.3?

Reply:
Yeah, not fully sure about this. So the difference is the exact url. The stable one links to https://pandas.pydata.org/docs/, while the other links to https://pandas.pydata.org/pandas-docs/version/1.3/. 

Currently, that's both linking to the exact same docs (for 1.3.5). But once 1.4.0 is out, the https://pandas.pydata.org/docs/ will start pointing to the 1.4 docs. 

So if we don't yet add this 1.3 specific entry, that means you can't (from this menu) get to a page / link that will always keep linking to the 1.3 docs. 
Now, that's maybe not too important to be able to do, so we could also leave it out (and only add the version-specific entry once the next stable release is out)

Reply:
Ah okay so it sounds like "latest released" vs a specific version.

I would be partial just to have links to specific versions and not necessarily https://pandas.pydata.org/docs/ but not a super strong opinion of mine.

Reply:
IMO I would include a link to the stable (https://pandas.pydata.org/docs/) docs. That is (or should be) the most used documentation version, and so I think it is good to have an explicit link to that from the older versions.

Reply:
Linking to the stable docs sounds good.